admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/service/impl/BulkDataUploadServiceImpl.java

```diff
@@ -388,13 +403,13 @@ private BulkDataResponseDto uploadPackets(MultipartFile[] files, String operatio
 			} catch (Throwable e) {
 				logger.error("Failed to sync and upload packet", e);
 				message = String.format(PKT_UPLOAD_MESSAGE, file.getOriginalFilename(), "FAILED", e.getMessage());
+				updateBulkUploadTransaction(bulkUploadTranscation, "FAILED");

```

Comment:
Since you are using FAILED string more than once, please create a static string and use it.